app/assets/stylesheets/homes.scss

```diff
@@ -0,0 +1,92 @@
+@import "utilities/variables";
+
+.valign-middle {
+    display: table;
+}
+
+.valign-middle .columns {
+    display: table-cell;
+    vertical-align: middle;
+}
+
+.valign-middle .columns,
+.valign-middle [class*="column"] + [class*="column"]:last-child {
+    float: none;
+}
+
+.top-bar {
+  background: url(../assets/top-bar-background.png) repeat-x;
+}
+
+.top-bar-section li:not(.has-form) a:not(.button) {
+  background: url(../assets/top-bar-background.png) repeat-x;
+}
+
+.top-bar-section ul li {
+  background: url(../assets/top-bar-background.png) repeat-x;
+}
+
+#nav-home {
+  font-family: 'Rock Salt', cursive;
+  span#hire-home {
```

Comment:
Rule declaration should be preceded by an empty line<br>Avoid using id selectors<br>Avoid qualifying id selectors with an element.